articles/app-service/includes/quickstart-custom-container/quickstart-custom-container-linux-azure-portal-pivot.md

```diff
@@ -0,0 +1,129 @@
+---
+author: cephalin
+ms.service: app-service
+ms.devlang: java
+ms.topic: quickstart
+ms.date: 06/30/2022
+ms.author: cephalin
+---
+
+[Azure App Service](../../overview.md) on Linux provides pre-defined application stacks on Linux with support for languages such as .NET, PHP, Node.js and others. You can also use a custom Docker image to run your web app on an application stack that isn't already defined in Azure. This quickstart shows you how to deploy an image from Azure Container Registry to Azure App Service.
+
+To complete this quickstart, you need:
+
+- An [Azure account](https://azure.microsoft.com/free/?utm_source=campaign&utm_campaign=vscode-tutorial-docker-extension&mktingSource=vscode-tutorial-docker-extension)
+- An [Azure container registry](/azure/container-registry/container-registry-get-started-portal)
+- [Azure CLI](/cli/azure/install-azure-cli)
+- [Docker](https://www.docker.com/community-edition)
+
+## 1 - Clone the sample repository
+
+We will clone the code so that we can update the code and then update our container.
+
+1. Go to [the .NET 6.0 sample app](https://github.com/Azure-Samples/dotnetcore-docs-hello-world).
+1. Select the **Code** button.
+1. Copy the HTTPS URL.
+1. In your command line, run:
+
+    ```bash
+    git clone https://github.com/Azure-Samples/dotnetcore-docs-hello-world.git
+    ```
+
+## 2 - Push the image to Azure Container Registry
+
+The cloned repository contains a **Dockerfile.linux** file.
+
+1. Log in to Azure Container Registry.
+
+    ```azurecli
+    az acr login -n <your_registry_name>
```

Comment:
you'll need az login first for this

also need to CD to the repo root

Reply:
Added